python/cudf/cudf/io/parquet.py

```diff
@@ -561,6 +567,116 @@ def read_parquet(
         **kwargs,
     )
 
+    # Apply filters row-wise (if any are defined), and return
+    return _apply_post_filters(df, filters)
+
+
+def _normalize_filters(filters: list | None) -> List[List[tuple]] | None:
+    # Utility to normalize and validate the `filters`
+    # argument to `read_parquet`
+    if filters is not None:
+        msg = (
+            f"filters must be None, or non-empty List[Tuple] "
+            f"or List[List[Tuple]]. Got {filters}"
+        )
+        if not filters or not isinstance(filters, list):
```

Comment:
Is it easier to accept empty containers as "no filters" and normalise them to `None` (rather than requiring specifically `None` as empty filters).

So:
```
if filters:
   ... validate
else:
   return None
```